packages/myst-directives/src/quote.ts

```diff
@@ -0,0 +1,78 @@
+import type { DirectiveSpec, DirectiveData, GenericNode } from 'myst-common';
+export const epigraphDirective: DirectiveSpec = {
+  name: 'epigraph',
+  doc: 'Inscriptions, or "epigraphs", provide a short quote or inscription at the beginning of a topic. They are usually pertinent to the subsequent content, either to set the theme, or establish a counter-example.',
+  options: {
+    label: {
+      type: String,
+      alias: ['name'],
+    },
+    class: {
+      type: String,
+      alias: ['figclass'],
+      doc: `CSS classes to add to your epigraph`,
+    },
+  },
+  body: {
+    type: 'myst',
+    doc: 'The body of the epigraph.',
+  },
+  run(data: DirectiveData): GenericNode[] {
+    const children: GenericNode[] = [];
+    if (data.body) {
+      children.push(...(data.body as GenericNode[]));
+    }
+
+    const container = {
+      type: 'container',
+      kind: 'quote',
+      class: 'epigraph',
```

Comment:
This should pick up the class and id/label from the options!

Reply:
There are some other examples in directives of doing this, the label and identifier need to be made here.